It fails to compile with errors:

| ../../../../sip/cpp/sip_gridwxGridEvent.cpp:35:9: error: 'int sipwxGridEvent::GetRow()'
  marked 'override', but does not override
|    35 |     int GetRow() SIP_OVERRIDE;
|       |         ^~~~~~
| ../../../../sip/cpp/sip_gridwxGridEvent.cpp:36:9: error: 'int sipwxGridEvent::GetCol()'
  marked 'override', but does not over      ride
|    36 |     int GetCol() SIP_OVERRIDE;
|       |         ^~~~~~

Make these functions non-override.

Upstream-Status: Pending

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 sip/cpp/sip_gridwxGridEvent.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sip/cpp/sip_gridwxGridEvent.cpp b/sip/cpp/sip_gridwxGridEvent.cpp
index 0830b677..314aec94 100644
--- a/sip/cpp/sip_gridwxGridEvent.cpp
+++ b/sip/cpp/sip_gridwxGridEvent.cpp
@@ -32,8 +32,8 @@ public:
      * this class.
      */
 protected:
-    int GetRow() SIP_OVERRIDE;
-    int GetCol() SIP_OVERRIDE;
+    int GetRow() ;
+    int GetCol() ;
      ::wxEvent* Clone() const SIP_OVERRIDE;
      ::wxEventCategory GetEventCategory() const SIP_OVERRIDE;
 
